src/fu-tool.c

```diff
@@ -2895,6 +2895,9 @@ fu_util_firmware_parse(FuUtilPrivate *priv, gchar **values, GError **error)
 		return FALSE;
 	}
 
+	/* match the behavior of the daemon as we're printing the children */
+	priv->parse_flags |= FU_FIRMWARE_PARSE_FLAG_CACHE_STREAM;
```

Comment:
(in the future It would be nice to add a unit test for the signing for coverage)